packages/sanity/src/core/create/components/CreateLinkedDocumentBannerContent.tsx

```diff
@@ -96,7 +96,8 @@ export function CreateLinkedDocumentBannerContent(props: CreateLinkedDocumentBan
           icon={InfoOutlineIcon}
           mode="bleed"
           onClick={toggleOpen}
-          style={{marginBottom: '-1em', marginTop: '-1em'}}
+          // Negative margins added to prevent the button from blowing out banner height
+          style={{marginBottom: '-0.5em', marginTop: '-0.5em'}}
```

Comment:
Incredibly minor tweak which just ensures the button is the correct size on hover.